stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/imagepreview/ImagePreviewActivity.kt

```diff
@@ -56,7 +56,7 @@ import java.util.Date
 public class ImagePreviewActivity : AppCompatActivity() {
 
     private val factory by lazy {
-        ImagePreviewViewModelFactory(ChatClient.instance())
+        ImagePreviewViewModelFactory(ChatClient.instance(), intent?.getStringExtra(KEY_MESSAGE_ID) ?: "")
```

Comment:
Does it make sense to run it with an empty message id? If not - maybe we should inform the user that something went wrong?